app/_src/policies/introduction.md

```diff
@@ -1,6 +1,650 @@
+```
+
+Here's an explanation of each kinds and their scope:
+
+- Mesh: applies to all proxies running in the mesh
+- MeshSubset: same as Mesh but filters only proxies who have matching `targetRef.tags`
+- MeshService: all proxies with a tag `kuma.io/service` equal to `targetRef.name`.{% if_version gte:2.9.x %} This can work differently when using [explicit services](#using-policies-with-meshservice-meshmultizoneservice-and-meshexternalservice){% endif_version %}.
+- MeshGateway: targets proxies matched by the named MeshGateway
+    - Note that it's very strongly recommended to target MeshGateway proxies using this
+      kind, as opposed to MeshService/MeshServiceSubset.
+- MeshServiceSubset: same as `MeshService` but further refine to proxies that have matching `targetRef.tags`. ⚠️This is deprecated from version 2.9.x ⚠️.
+
+Consider the example below:
+
+{% policy_yaml accesslog_example %}
+```yaml
+type: MeshAccessLog
+name: example
+mesh: default
+spec:
+  targetRef: # top level targetRef
+    kind: MeshSubset
+    tags:
+        app: web-frontend
+  to:
+    - targetRef: # to level targetRef
+        kind: MeshService
+        name: web-backend
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+  from:
+    - targetRef: # from level targetRef
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+```
+{% endpolicy_yaml %}
+
+Using `spec.targetRef`, this policy targets all proxies that have a tag `app:web-frontend`.
+It defines the scope of this policy as applying to traffic either from or to dataplane proxies with the tag `app:web-frontend`.
+
+The `spec.to[].targetRef` section enables logging for any traffic going to `web-backend`.
+The `spec.from[].targetRef` section enables logging for any traffic coming from _anywhere_ in the `Mesh`.
+
+### Omitting `targetRef`
+When a `targetRef` is not present. It is semantically equivalent to: `targetRef.kind: Mesh` meaning everything inside the `Mesh`.
+
+### Applying to specific proxy types
+The `targetRef` field can select a specific subset of data plane proxies. The field named `proxyTypes` can restrict policies to specific types of data plane proxies:
+- `Sidecar`: Targets data plane proxies acting as sidecars to applications.
+- `Gateway`: Applies to data plane proxies operating in Gateway mode.
+- Empty list: Defaults to targeting all data plane proxies.
+
+#### Example
+
+The following policy will only apply to gateway data-planes:
+{% policy_yaml proxytypes %}
+```yaml
+type: MeshTimeout
+name: gateway-only-timeout
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: ["Gateway"]
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        idleTimeout: 10s
+```
+{% endpolicy_yaml %}
+
+### Target resources
```

Comment:
I don't understand this heading, and after reading the paragraph it don't make much more sense to me, should it be something like `TargetRef kinds supported in different policies`?